taskswamp

```diff
@@ -85,9 +85,14 @@ def main():
 
     assert config['taskswamp'] == -1, "Missing config format number"
 
+    refresh_every = "0"
+    if 'refresh_every' in config:
+        refresh_every = config['refresh_every']
+    assert int(refresh_every) >= 0
```

Comment:
Secondarily, more as a matter of coding style: it would be more idiomatic to write `refresh_every = config.get('refresh_every', 0)`, i.e., to use `.get()` and to keep the value as a numeric type internally.

Reply:
fixed in 2e982473